projects/services-layers/src/lib/types/Layers.ts

```diff
@@ -79,13 +81,20 @@ export const Filtertypes = {
 export type TFiltertypes = keyof typeof Filtertypes;
 
 
-
+/**
+ * CustomLayertype and TmsLayertype can be raster and vector.
+ * You have to double check by yourself later!
```

Comment:
Maybe we have to recheck this with `isRasterLayer()` so that no one expects to test for raster layers and get a false positive.

looks it is only used in:
- layerentry.component: https://github.com/dlr-eoc/ukis-frontend-libraries/search?q=isVectorLayer
- owc-json.service: https://github.com/dlr-eoc/ukis-frontend-libraries/search?q=isVectorLayertype

Reply:
maybe mark it for now as deprecated and remove these methods later on. It should be a user responsibility to determine which layer it is

Reply:
ccc4e58b0ffa3e08c61f9cd36755363b9252772a